### PR TITLE
[REFACTOR] removed two unnecessary package

### PR DESCRIPTION
**Pull request by murshadahammed**
Removed :
     -multiselect
     -multiselect-react-dropdown

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,8 +18,7 @@
     "react-scripts": "5.0.1",
     "react-select": "^5.7.0",
     "react-to-print": "^2.14.10",
-    "styled-components": "^5.3.6",
-    "web-vitals": "^2.1.4"
+    "styled-components": "^5.3.6"
```
Comment (sharan-experion) [inline package.json:21]: This package is also unused.

Comment (murshadahammed) [reply package.json:21]: styled components are used for data-table structuring.. right?

Comment (sharan-experion) [reply package.json:21]: could you tell me where it is used in the project? I need to take a look.

Comment (murshadahammed) [reply package.json:21]: D:\automotive\Automotive_Revamped\src\pages\EmployeeSheetDisplay\EmployeeSheetDisplay.js

**Review by sharan-experion — APPROVED** [review]

**Pull request merged**